### PR TITLE
fix: replace h tags

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

Tags are being read wrong by Crowdin hidden string process. Not sure why
challenge text is using headers anyway - replaced with bold/strong.

Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

cc/ @RandellDawson

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/24-game.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The aim of the game is to arrange four numbers in a way that when evaluated, the
 
 Implement a function that takes a string of four digits as its argument, with each digit from 1 to 9 (inclusive) with repetitions allowed, and returns an arithmetic expression that evaluates to the number 24. If no such solution exists, return "no solution exists".
 
-<h4><strong>Rules:</strong></h4>
+<strong>Rules:</strong>
```
Comment (RandellDawson) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/24-game.md:19]:
```suggestion
**Rules:**
```

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/balanced-brackets.md [hunk 2]
```diff
@@ -10,8 +10,7 @@ dashedName: balanced-brackets
 
 Determine whether a generated string of brackets is balanced; that is, whether it consists entirely of pairs of opening/closing brackets (in that order), none of which mis-nest.
 
-<h4><strong>Examples:</strong></h4>
-
+<strong>Examples:</strong>
```
Comment (RandellDawson) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/balanced-brackets.md:13]:
```suggestion
**Examples:**
```

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/hash-join.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ In pseudo-code, the algorithm could be expressed as follows:
 
 Implement the "hash join" algorithm as a function and demonstrate that it passes the test-case listed below. The function should accept two arrays of objects and return an array of combined objects.
 
-<h4><strong>Input</strong></h4>
+<strong>Input</strong>
```
Comment (RandellDawson) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/hash-join.md:44]:
```suggestion
**Input**
```

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/hash-join.md [hunk 4]
```diff
@@ -127,7 +127,7 @@ Implement the "hash join" algorithm as a function and demonstrate that it passes
   </tr>
 </table>
 
-<h4><strong>Output</strong></h4>
+<strong>Output</strong>
```
Comment (RandellDawson) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/hash-join.md:130]:
```suggestion
**Output**
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@nhcarrigan I made 4 suggestions (to use markdown syntax for the `<strong>`.

**Comment by nhcarrigan** [comment]
Thanks 🙂

**Review by RandellDawson — APPROVED** [review]
LGTM

**Comment by nhcarrigan** [comment]
Not sure why I failed a Euler Challenge test, but I'll try re-running the job.

**Pull request merged**